persistence/src/androidMain/kotlin/com/wire/kalium/persistence/db/UserDatabase.kt

```diff
@@ -75,6 +79,19 @@ actual fun userDatabaseBuilder(
     return UserDatabaseBuilder(userId, driver, dispatcher, platformDatabaseData, passphrase != null)
 }
 
+actual fun userDatabaseDriver(
+    platformDatabaseData: PlatformDatabaseData,
+    dbPath: String
+): SqlDriver {
+    System.loadLibrary("sqlcipher")
+    val db: SupportSQLiteDatabase = net.zetetic.database.sqlcipher.SQLiteDatabase.openDatabase(
+        dbPath,
+        null,
+        SQLiteDatabase.OPEN_READWRITE
+    )
+    return AndroidSqliteDriver(db, 100)
```

Comment:
the 100 is cache size i was not able to find and documentation this value and it can not be 0